src/spacefinder/liveblog-adverts.ts

```diff
@@ -31,34 +31,28 @@ const AD_SPACE_MULTIPLIER = 2;
 
```

Comment:
Bear in mind the changes in this file are to allow inserting ads client side and server side both to work, depending on whether DCR inserts slots server-side or not. Once this PR is merged, the DCR PR to insert slots server-side will follow, then another PR to remove the client-side slot insertion functionality from this file will be removed.